web/templates/home.html.tmpl

```diff
@@ -1,71 +1,5 @@
-                        host accordingly.
-                    </p>
-                    <div class='d-flex justify-content-end'>
-                        <a id='tn-hosts-link' class='card-link' href='/hosts'>Go to view</a>
-                    </div>
-                </div>
-            </div>
-            <div class='card'>
-                <h5 class='card-header'>Pacemaker Clusters</h5>
-                <div class='card-body'>
-                    <p class='card-text'>
-                        This view currently allows to check the status of the all the discovered
-                        Pacemaker clusters, including the information of the status of each node and
-                        the role associated to each node. This view also allows to see the node's
-                        attributes which reflect details found from their configuration as well as
-                        resources, their type and distribution within the cluster nodes.
-                    </p>
-                    <div class='d-flex justify-content-end'>
-                        <a id='tn-clusters-link' class='card-link' href='/clusters'>Go to view</a>
-                    </div>
-                </div>
-            </div>
-        </div>
-        <div class='card-deck mt-4'>
-            <div class='card'>
-                <h5 class='card-header'>SAP Systems</h5>
-                <div class='card-body'>
-                    <p class='card-text'>
-                        In this view are listed the SAP Systems, usually identified by a SID or
-                        `SAP System Identification` string such as `PRD`, `DEV`, or `QAS`. These are used
-                        often to identify productive, development and quality assurance systems.
-                        In this view we get an overview of the distribution of each SID, the number of
-                        hosts discovered of each of them and additional details.
-                    </p>
-                    <div class='d-flex justify-content-end'>
-                        <a id='tn-sapsystems-link' class='card-link' href='/sapsystems'>Go to view</a>
-                    </div>
-                </div>
-            </div>
-        </div>
-        <div class='mt-4'>
-            For more information and getting started, please refer to the
-            <a id='tn-scope-link' target='blank'
-               href='https://github.com/trento-project/trento/blob/main/docs/scope.md'>scope</a>,
-            <a id='tn-readme-link' target='blank'
-               href='https://github.com/trento-project/trento/blob/main/README.md'>readme</a>
-            or
-            <a id='tn-architecture-link' target='blank'
-               href='https://github.com/trento-project/trento/blob/main/docs/trento-architecture.md'>architecture</a>
-            documents.
-        </div>
-    </section>
+  <div id="homepage-component"></div>
```

Comment:
Question for @abravosuse : Do you think we should move the current home content to somewhere else?
Like the about page or some additional page. Just asking.
It doesn't need to done in this PR anyway

Reply:
Yes, I think that eventually we will have to move it somewhere else because the home page should all be about the status of your environment. But I'd rather keep the About page as it it. That way the version info is clearly displayed and there is no confusion about it. Maybe another page under Settings called "Intro" or "First steps". To be refined as well.